dsctl_tls_test.py fails with openssl-3.x

**Issue Description**
openssl-3.x has changed an error message.
openssl-1.1.1n-1.fc35.x86_64:
```
unable to load private key
139720888094208:error:0909006C:PEM routines:get_name:no start line:crypto/pem/pem_lib.c:745:Expecting: ANY PRIVATE KEY
```

openssl-3.0.2-5.fc36.x86_64:
```
Warning: output encryption option -aes128 ignored with -export
Could not read private key from -inkey file from /etc/dirsrv/slapd-standalone1/dse.ldif
```

Because of that dsctl_tls_test.py fails on Fedora 36 that has openssl-3.x:

```
>           assert 'unable to load private key' in str(e)
E           AssertionError: assert 'unable to load private key' in 'Warning: output encryption option -aes128 ignored with -export\nCould not read private key from -inkey file from /etc/dirsrv/slapd-standalone1/dse.ldif'
E            +  where 'Warning: output encryption option -aes128 ignored with -export\nCould not read private key from -inkey file from /etc/dirsrv/slapd-standalone1/dse.ldif' = str(ValueError('Warning: output encryption option -aes128 ignored with -export\nCould not read private key from -inkey file from /etc/dirsrv/slapd-standalone1/dse.ldif'))
```

--- a/dirsrvtests/tests/suites/clu/dsctl_tls_test.py
+++ b/dirsrvtests/tests/suites/clu/dsctl_tls_test.py
@@ -1,5 +1,6 @@
 import logging
 import pytest
+import ssl
 import os
 from lib389.topologies import topology_st as topo
 from lib389.nss_ssl import NssSsl
@@ -69,7 +70,10 @@ def test_tls_command_returns_error_text(topo):
         assert False
     except ValueError as e:
         assert '255' not in str(e)
-        assert 'unable to load private key' in str(e)
+        if 'OpenSSL 3' in ssl.OPENSSL_VERSION:
+            assert 'Could not read private key from' in str(e)
+        else:
+            assert 'unable to load private key' in str(e)
 
 
 if __name__ == '__main__':

